### DISCUSSION TITLE
can i get the step execution result  in github actions

### DISCUSSION THREAD

**Discussion by zl-go** [community/community#37285] [Actions]
i wanna push some files in github actions timely,but in case some terrible thing happens,i want to get the result to inform myself by emails.

**Comment by airtower-luna** [comment]
There's nothing built-in, but if you want to receive an email with the step output you can add another step that sends the email (there are custom actions to do this, but I can't recommend a particular one). Note that you'll need to provide a mail account for that.

You can use the [status check functions](https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions) to easily check if a previous step failed or was canceled.

**Comment by ossanna16** [comment]
Hi there @zl-go and welcome to our community! Thank you for asking a great question 🙂
